staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -31,171 +31,278 @@ import (
 	"k8s.io/apimachinery/pkg/util/json"
 )
 
-func getNestedField(obj map[string]interface{}, fields ...string) interface{} {
+// NestedField returns the value of a nested field.
+// false is returned if the value is missing.
```

Comment:
So `null` gives `nil, true`, undefined gives `nil, false` ? Maybe be a bit more verbose here.